meta/main.yml

```diff
@@ -14,6 +14,14 @@ galaxy_info:
       versions:
         - trusty
         - xenial
+    - name: EL
```

Comment:
Don't change yet.

We'll update it later on final stage, after this PR, `st2web` PR and after disabling `allow_failures` for CentOS6/7 CI: https://github.com/StackStorm/ansible-st2/blob/master/.travis.yml#L14